lms/envs/aws.py

```diff
@@ -1183,3 +1183,8 @@
     'RATE_LIMIT_BACKEND_MAX_REQUESTS',
     RATE_LIMIT_BACKEND_MAX_REQUESTS
 )
+
+############### Settings for Assets access by using an access token ###############
+ASSETS_ACCESS_BY_TOKEN = ENV_TOKENS.get('ASSETS_ACCESS_BY_TOKEN', ASSETS_ACCESS_BY_TOKEN)
```

Comment:
@naeem91 These tokens should be fetched from `AUTH_TOKENS` dict as `lms.auth.json` contains all the secret tokens. Please make desired changes in `mckinsey_internal` repo also

Reply:
@ihtram Suggested changes made. I'll open a PR for internal and also request for placing actual key in secure repo.

Reply:
@naeem91 Looks good now. I am approving this PR, please squash commits and open a PR for internal repo.